platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/task/AGWorkflowTask.java

```diff
@@ -53,4 +53,14 @@ public boolean wasNotExecuted() {
 			.where(AGWorkflowTransitionExecution.WORKFLOW_TASK.isEqual(this))
 			.exists();
 	}
+
+	public Optional<AGWorkflowTaskDelegation> getDelegation() {
+
+		return Optional.ofNullable(AGWorkflowTaskDelegation.TABLE.get(this));
```

Comment:
Having a public method that may return an inactive delegation seems odd (like a trap) to me. Can we change this to `Optional<AGWorkflowTaskDelegation> getActiveDelegation()`?

Reply:
done